### ISSUE TITLE
Remove the outer div in the datepicker

### ISSUE THREAD

**Issue by arthurdenner** [arthurdenner/react-semantic-ui-datepickers#19]
# 🌈 Feature

Right now, we have an outer `div` in the datepicker with the `field` class wrapping the `Input` component and the calendar. This approach has some issues, especially related to how the library behaves in forms.

We must remove this `div` and implement the rendering of the calendar in another way. I'm not sure how this is supposed to work, but maybe these implementations can help:

https://github.com/CompuIves/codesandbox-client/blob/master/packages/app/src/app/components/ContextMenu/index.js

https://github.com/CompuIves/codesandbox-client/blob/master/packages/app/src/app/components/Portal/index.js

**Comment by arthurdenner** [comment]
@lucasnsborges, you may want to take a stab at this.

**Comment by erickne** [comment]
I'm probably having this issue in a form inside a modal. 
![image](https://user-images.githubusercontent.com/1703652/50831060-db4b1680-1330-11e9-801c-2fcce77434f9.png)

Unfortunatelly, I to use this workaround to fix.
```
input {
    width: inherit !important;
}
```

**Comment by arthurdenner** [comment]
Yes, @erickne. This is probably the reason. Sorry about that. :confused:

If you want to take a stab at this, feel free to do it, I'd appreciate a lot!
